kernel/rtlil.h

```diff
@@ -1216,6 +1224,28 @@ struct RTLIL::Module : public RTLIL::AttrObject
 #endif
 };
 
+struct RTLIL::Parameter : public RTLIL::AttrObject
+{
+	unsigned int hashidx_;
+	unsigned int hash() const { return hashidx_; }
+
+protected:
+	friend struct RTLIL::Module;
+	Parameter();
+	Parameter(RTLIL::Const defaultValue);
+
+	RTLIL::IdString	name;
+	RTLIL::Const	defaultValue;
+
+public:
+
+	RTLIL::Const getDefaultValue () const { return defaultValue; }
+
+//#ifdef WITH_PYTHON
```

Comment:
I'm pretty sure we want the parameters to be available in Python?

Reply:
Yep. I know that it has to be enabled in the end.

Reply:
I've been looking how the python interface to Yosys is built and to be honest I do not have a clue how it is done. There are some functions surrounded by `#ifdef WITH_PYTHON` but I don't see how they are mapped to a Python API. So I haven't added any Python support for accessing attributes of parameters yet.

Reply:
I figured it out.